### ISSUE TITLE
Crystal sometimes crashes when removing other apps

### ISSUE THREAD

**Issue by sliceofcode** [sliceofcode/crystal#1]

**Comment by sliceofcode** [comment]
I can't seem to reproduce this one, but I have definitely seen it happen before.

**Comment by sliceofcode** [comment]
```java
09-14 17:25:46.903 14130 14130 E AndroidRuntime: FATAL EXCEPTION: main
09-14 17:25:46.903 14130 14130 E AndroidRuntime: Process: ws.slice.crystal, PID: 14130
09-14 17:25:46.903 14130 14130 E AndroidRuntime: java.lang.RuntimeException: Unable to start receiver ws.slice.crystal.UninstallReceiver: java.lang.NullPointerException: Attempt to read from field 'java.io.File ws.slice.crystal.VersionsActivity.mNextBackedUpApkToInstall' on a null object reference
09-14 17:25:46.903 14130 14130 E AndroidRuntime: 	at android.app.ActivityThread.handleReceiver(ActivityThread.java:3642)
09-14 17:25:46.903 14130 14130 E AndroidRuntime: 	at android.app.ActivityThread.access$2000(ActivityThread.java:222)
09-14 17:25:46.903 14130 14130 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1877)
09-14 17:25:46.903 14130 14130 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:102)
09-14 17:25:46.903 14130 14130 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:158)
09-14 17:25:46.903 14130 14130 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:7229)
09-14 17:25:46.903 14130 14130 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
09-14 17:25:46.903 14130 14130 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
09-14 17:25:46.903 14130 14130 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)
09-14 17:25:46.903 14130 14130 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to read from field 'java.io.File ws.slice.crystal.VersionsActivity.mNextBackedUpApkToInstall' on a null object reference
09-14 17:25:46.903 14130 14130 E AndroidRuntime: 	at ws.slice.crystal.UninstallReceiver.onReceive(UninstallReceiver.java:24)
09-14 17:25:46.903 14130 14130 E AndroidRuntime: 	at android.app.ActivityThread.handleReceiver(ActivityThread.java:3635)
09-14 17:25:46.903 14130 14130 E AndroidRuntime: 	... 8 more
```

Got it.

**Comment by sliceofcode** [comment]
Fixed with `914fc08`.
